modules/cfe_testcase/src/cfe_test.h

```diff
@@ -46,6 +46,7 @@
 typedef struct
 {
     CFE_FS_FileWriteMetaData_t FuncTestState;
+    int                        count;
```

Comment:
Recommend either a more useful name or add a comment as to what the intent for this variable is (is it intended as a generic utility global or specifically for use in the es task test?) another improvement would be consistent case (camel).   Helpful also to have `extern CFE_FT_Global_t CFE_FT_Global` in the header, and instantiate it once.